lib/java/com/google/android/material/transition/MaterialContainerTransform.java

```diff
@@ -935,11 +936,11 @@ public Animator createAnimator(
     final TransitionDrawable transitionDrawable =
         new TransitionDrawable(
             getPathMotion(),
-            startView,
+            copyViewImage(sceneRoot, startView, startView.getParent() instanceof View ? (View) startView.getParent() : startView),
```

Comment:
I think this should become `startView.getParent() instanceof View ? copyViewImage(sceneRoot, startView, (View) startView.getParent()) : startView`

Maybe better to extract this to a function.
```
  private static View maybeGetViewImageCopy(ViewGroup sceneRoot, View view) {
    if (view.getParent() instanceof View) {
      return TransitionUtils.copyViewImage(sceneRoot, view, (View) view.getParent());
    }
    return view;
  }
  ```